archaius-core/src/main/java/netflix/archaius/PropertyConversions.java

```diff
@@ -0,0 +1,66 @@
+package netflix.archaius;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+/**
+ * Interface for converting properties between various types.
+ */
+public interface PropertyConversions {
```

Comment:
Having this functionality in ObservableProperty feels conflated to me, as it has basically nothing to with a property being Observable.  Another option would be to put it directly in the Property interface but that seemed like it might be overly strict.